### DISCUSSION TITLE
No way to filter issues using mouse

### DISCUSSION THREAD

**Discussion by rafrafek** [community/community#123245] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

I am very unhappy with the current state of the Issues page. There is no button to submit the text field.

I always paste something using my mouse. I want to submit it. There is no filter/submit button.

I have to buy a keyboard, connect it to my computer, and hit Return on the keyboard.

<img width="1235" alt="image" src="https://github.com/community/community/assets/23004737/52096d12-9306-4bcd-8fa6-5800fa007cf5">

Other companies do it much better.

For example here I can paste something and click Google Search button with my mouse and it will work without any keyboard connected to my computer.

<img width="733" alt="image" src="https://github.com/community/community/assets/23004737/e6ac4ea6-941f-40eb-bc2d-bfa21a239c0c">

Please send this link to your web programmers so they can read more about how to implement a html button. It may look difficult but as I said it was achieved by other tech companies like Google.

https://www.w3schools.com/tags/tag_button.asp

Thank you for your understanding. I know I'm asking for something very hard to do, but I believe mouse users should not be left without the possibility to filter issues on GitHub.

**Comment by rafrafek** [comment]
Sadly mouse users are facing discrimination on this platform. The issue is not new https://github.com/orgs/community/discussions/34705

**Comment by rafrafek** [comment]
Sadly @steveward already answered:

>When thinking through accessibility the focus is generally on ensuring that websites do not require a mouse, and can be fully navigated with a keyboard.

https://github.com/github/docs/pull/21005#issuecomment-1266115614

Which means discrimination is OK when the mouse users are victims. Only the keyboard users are protected.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Comment by cehfisher** [comment, marked as answer]
Hi @rafrafek - this issue looks like a [duplicate](https://github.com/orgs/community/discussions/34705) of one you have already filed and that [we responded to recently](https://github.com/orgs/community/discussions/34705#discussioncomment-9539964). Could you please close this duplicate issue, so we can focus on responding to your request in one location? Thank you!
